Ifzan/linear.js

```diff
@@ -0,0 +1,28 @@
+function line(arr,find)
+{
+    if(Array.isArray(arr) !== true || arr == '')
+    {
+        console.log('Invalid');
+        return;
+    }
+    else
+    {
+        let len = arr.length;
+        for(let i=0; i<len; i++)
+        {
+            let a =  0;
```

Comment:
you can simplify this without the need of another variable 'a'

Reply:
break out of loop, if you find

Reply:
tried and executed successfully.